.travis_setup.sh

```diff
@@ -2,23 +2,24 @@
 
 set -eux
 
-CHANGED_PWD=false
-# Change the password recreating the root user on mariadb < 10.2
+# Change the password to be empty.
+CHANGED_PASSWORD=false
+# Change the password to be empty, recreating the root user on mariadb < 10.2
 # where ALTER USER is not available.
 # https://stackoverflow.com/questions/56052177/
-CHANGED_PWD_BY_RECREATE=false
+CHANGED_PASSWORD_BY_RECREATE=false
```

Comment:
* I renamed `CHANGED_PWD` to `CHANGED_PASSWORD`, because `PWD` makes me imagine `pwd` (print working directory) command. Sorry it was my typo.